nautobot/core/api/views.py

```diff
@@ -201,6 +210,8 @@ def get_serializer(self, *args, **kwargs):
 
         return super().get_serializer(*args, **kwargs)
 
+    # TODO: need to re-enable the below. Are the changes here actually part of the UI work or are these actually
+    # part of the "reworking REST API nested serializers" proof-of-concept?
```

Comment:
The nested-serializer stuff is, we think, a prerequisite for the automatic form generation stuff.

Reply:
It absolutely is. I created this to track that: https://github.com/nautobot/nautobot/issues/3042